docs/troubleshoot/cli/use-individual-troubleshoot-commands.md

```diff
@@ -61,13 +61,12 @@ The values for these variables can be **echo**ed.
 
 ![Home Directory](../../images/troubleshoot/cli/home_struc.png)
 
-### Location of logs
+### Location and types of logs
 
-There are two sets of logs to be aware of:
+Zowe client logs are located in the `~/.zowe/logs` directory:
 
-- Imperative CLI Framework log, which generally contains installation and configuration information.
-
-- Zowe CLI log, which contains information about interaction between CLI and the server endpoints.
+- **Imperative CLI Framework log**, which generally contains installation and configuration information.
```

Comment:
It might be somewhat obvious, but I would add the filenames to make it clear that the Imperative CLI Framework logs are stored in imperative.log and the Zowe CLI logs are stored in zowe.log.